src/clinical/clinical-service.ts

```diff
@@ -44,23 +44,33 @@ import { runTaskInWorkerThread } from './service-worker-thread/runner';
 
 const L = loggerFor(__filename);
 
-export type ClinicalQuery = {
+// Base type for Clinical Data Queries
+export type ClinicalDonorEntityQuery = {
   programShortName: string;
-  page: number;
-  pageSize?: number;
-  entityTypes: Array<EntityAlias>;
-  sort: string;
   donorIds: number[];
   submitterDonorIds: string[];
+  entityTypes: EntityAlias[];
   completionState?: {};
 };
 
-export type ClinicalSearchQuery = {
+// Returns paginated Donor + Entity Submission Data
+export interface PaginatedClinicalQuery extends ClinicalDonorEntityQuery {
+  page: number;
+  pageSize?: number;
+  sort: string;
+}
+
+// GQL Query Arguments
```

Comment:
Added new types for GQL request variables

Reply:
I think `interface extends type` is incorrect usage.
The way types extend is usually by unions.
I'm not sure why TS isn't throwing an error here.
Usually the main reason to use an interface is to extend from an interface.

What's the reasoning using the `interface` vs a `type`?
I ask because most of our code is written in types based on some talk we had a while back on slack with preferring types.

Reply:
Aha I see, still working on my Typescript. So where I got confused is `Type extends` is not a thing, `Type & {}` is the alternative. I used `interface` b/c I wanted to `extend` and didn't revisit.